test/libsolidity/syntaxTests/nameAndTypeResolution/invalidArgs/creating_struct_members_skipped.sol

```diff
@@ -7,3 +7,4 @@ contract C {
 }
 // ----
 // TypeError: (117-126): Wrong argument count for struct constructor: 1 arguments given but expected 2. Members that have to be skipped in memory: c
+// TypeError: (104-114): Type struct C.S memory is only valid in storage.
```

Comment:
Can you use the "short" variant of the 'toString' method here (which does not add `memory`)?